### DISCUSSION TITLE
Push docker images to docker hub / github container registry

### DISCUSSION THREAD

**Discussion by felbinger** [abraunegg/onedrive#1401] [Q&A] [answered]
It would be nice if you share the build images for all architectures and flavors, checkout the repository of https://github.com/MarcelCoding/onedrive-docker
He added a github action to build and push these images to https://ghcr.io and https://hub.docker.com

**Comment by abraunegg** [comment]
@felbinger 
Not an issue, should be a discussion. 

Please follow the correct support process as detailed here:
* https://github.com/abraunegg/onedrive#have-a-question
* https://github.com/abraunegg/onedrive#reporting-an-issue-or-bug

**Comment by abraunegg** [comment]
@felbinger 
Perhaps @MarcelCoding could initiate a PR request so that this could be merged in? Happy to work with folk who can contribute to improving the solution. 

Additionally, Docker images are currently available here: [Docker Hub](https://hub.docker.com/r/driveone/onedrive )

**Reply by MarcelCoding** [reply]
@abraunegg Ok, I'll provide a pull request in the next few days.

Do you want the Docker images also to be published to the github container regestry? - If yes, then enable it please. 
Could you please provide me the names of you secrets for the Docker hub username and Docker hub token?

And could you have a look to the tag names if they are as you expected?

**Comment by MarcelCoding** [comment]
@abraunegg -> https://github.com/abraunegg/onedrive/discussions/1401#discussioncomment-599071

**Comment by abraunegg** [comment, marked as answer]
@felbinger 
With the release of v2.4.18 - Docker containers are being automatically built and pushed to Docker Hub. Please check [docker.md](https://github.com/abraunegg/onedrive/blob/master/docs/Docker.md) for updated Docker Tag details.
